sql/moz-fx-data-shared-prod/braze_external/dataset_metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Braze External
+description: |-
+  Dataset with tables that sync to Braze.
+  These tables are ingested into Braze via currents.
+dataset_base_acl: restricted
+user_facing: false
+labels: {}
+workgroup_access:
+- role: roles/bigquery.dataViewer
```

Comment:
Maybe worth checking with Data SRE, but shouldn't this look more like this:
```
workgroup_access:
- role: roles/bigquery.dataViewer
   members:
   - workgroup:braze
- role: roles/bigquery.dataWriter
   members:
   - workgroup:braze/writers
```

?

Reply:
I found an example:
https://github.com/mozilla/bigquery-etl/blob/8323733162134c27b6207080059a8f7ea49263e8/sql/moz-fx-data-shared-prod/search_terms_derived/sanitization_job_metadata_v2/metadata.yaml#L17-L23

Reply:
Kik's correct. We want read access for `workgroup:braze` and write access for `workgroup:braze/writers`.

Reply:
thanks @kik-kik & @whd! i've pushed those changes :)